pkg/components/accesslog.go

```diff
@@ -90,12 +95,14 @@ func AccessLog(_ context.Context, _ string, _ string, _ string) (interface{}, er
 
 // Settings generates a config populated with defaults.
 func (m *AccessLogComponent) Settings() *AccessLogConfig {
-	return &AccessLogConfig{}
+	return &AccessLogConfig{
+		PrincipalHeader: "X-Slauth-Subject",
```

Comment:
If the concern was using our internal headers in this repo, maybe it's not ideal to have this as the default? I'm not totally opposed to it, but it feels a little weird.

Reply:
That's fair. I wanted some default so that we weren't forcing people to configure this if they didn't care about it. I also didn't want a default that could actually work (like JWT_SUBJECT or whatever) that could have users accidentally logging principals if they didn't want them.

I'll switch to something like DEFAULT-PRINCIPAL-HEADER to be sure that it's opt in logging.

Reply:
You can always make up headers like `X-Principal` or `X-Username` to use a placeholder. Alternatively, you can leave the default as empty string and it will reliably select no header value. It would be rare to get a header with no name and it may even be against the spec in [rfc2822](https://tools.ietf.org/html/rfc2822#section-2.2).

Either way, I'll need to override whatever default value you choose with my own relevant header so it doesn't affect me one way or another.